src/shared/prerendered-app/util.css

```diff
@@ -0,0 +1,24 @@
+:global {
```

Comment:
Making these global simplifies the build, and reduces duplication in the CSS

Reply:
Are all `.css` files presumed to be ICSS modules by default? I've been seeing most folks move to only treating `.module.css` files as "css modules", which feels nice.